tests/test_extend_schema.py

```diff
@@ -179,6 +181,24 @@ def non_required_body(self, request):
         def document(self, request):
             return Response(b'deadbeef', status=200)  # pragma: no cover
 
+        @extend_schema(
```

Comment:
since this is now part of `test_examples.py` if can be safely removed as it should be covered there completely